src/main/java/www/purple/mixxy/dao/ComicDao.java

```diff
@@ -15,6 +15,7 @@
 import www.purple.mixxy.models.Comic;
 import www.purple.mixxy.models.ComicDto;
 import www.purple.mixxy.models.ComicsDto;
+import www.purple.mixxy.models.Like;
 import www.purple.mixxy.models.User;
 
 public class ComicDao {
```

Comment:
Can you clarify what happens with these methods if no data is found (e.g. no parent, a bad parameter, no tags, etc.)?  I'm *assuming* everything that returns a `Comic` would return `null`, and everything that returns a `List` would return an empty list...but if not, oops.